source/_static/js/main.js

```diff
@@ -504,4 +504,12 @@ window.addEventListener("DOMContentLoaded", (event) => {
       });
     }
   })();
-});
\ No newline at end of file
+});
```

Comment:
```suggestion
// ------------------------------------------------
// Tab switching for hidden anchors
// ------------------------------------------------

    document.getElementById((document.getElementById(window.location.hash.substring(1)))
        .closest("div")
        .previousSibling.getAttributes("for"))
    .click();

});
```

Reply:
Maybe? Not sure if I got the spaces correct.

Reply:
I am but a lowly C programmer, but I will apply this and see if it works.

Reply:
@ravindk89 fresh stage with this change

Reply:
Hm. We're missing something here, because now the scrolldown doesn't trigger at all. Something in this logic must be interrupting the anchor scroll.

There is a `scrollIntoView()` function but I'm not sure how to apply it here.